investment-app/TRINV.Api/TRINV.Application/Commands/NewsComment/UpdateNewsCommentCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace TRINV.Application.Commands.NewsComment;
+
+using System.ComponentModel.DataAnnotations;
+using Domain.Entities;
+using Interfaces;
+using MediatR;
+using Shared.Business.Exceptions;
+using Shared.Business.Extension;
+using Shared.Business.Utilities;
+
+public record UpdateNewsCommentCommand : IRequest<OperationResult>
+{
+    [Required]
+    public int NewsCommentId { get; set; }
+
+    [Required]
+    [MaxLength(200)]
```

Comment:
On create your maxLength is 255 here is 200. Which should be ?

Reply:
Done.